src/WebJobs.Script.WebHost/Controllers/SwaggerController.cs

```diff
@@ -13,7 +13,7 @@
 
 namespace Microsoft.Azure.WebJobs.Script.WebHost.Controllers
 {
-    [SystemAuthorizationLevel(ScriptConstants.SwaggerDocumentationKey)]
```

Comment:
Merging this behavior into the base attribute where it makes more sense, and we avoid duplication